word_way/word_data.py

```diff
@@ -0,0 +1,103 @@
+""":mod:`word_way.word_data` --- 단어 정보 저장(DB)과 관련된 API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+import typing
+import xml.etree.ElementTree as elemTree
+
+import requests
+from flask import Blueprint, abort, jsonify, request
+
+from .config import get_word_api_config
+from .models import Word, Pronunciation, Sentence, WordSentenceAssoc
+from .context import session
+from .utils import convert_word_part
+
+__all__ = 'word_data', 'save_word_data'
+
+
+word_data = Blueprint('word_data', __name__, url_prefix='/word_data')
+
+
+@word_data.route('/save/', methods=['POST'])
+def save_word_data():
+    '''우리말샘 API로 단어 정보를 가져와서 DB에 저장하는 API'''
+    params = request.get_json()
+    target_word = params.get('word')
+
+    # 단어 기본 정보 요청
+    config = get_word_api_config()
+    params = {
+        'key': config.get('token'),
+        'q': target_word,
+        'target_type': 'search',
+        'part': 'word',
+        'sort': 'dict',
+    }
+    res = requests.get(config.get('url') + 'search', params=params)
+    res_tree = elemTree.fromstring(res.text)
+
+    words = save_word(target_word, res_tree)
+    if not words:
+        abort(404)
```

Comment:
```suggestion
        raise NotFound('sadf')
```
1. 저는 직관적인 `NotFound`, `BadRequest`를 raise하는 것을 좀 더 선호하긴 합니다.
2. 에러의 이유도 적어주세요.